src/ReturnTypes/AppMakeHelper.php

```diff
@@ -41,6 +43,11 @@ public function resolveTypeFromCall(FuncCall|MethodCall|StaticCall $call, Scope
                     $resolved = $this->resolve($constantString->getValue());
 
                     if ($resolved === null) {
+                        if (class_exists($constantString->getValue()) || interface_exists($constantString->getValue())) {
```

Comment:
We can't use `class_exists` We need to inject ` PHPStan\Reflection\ReflectionProvider` and ask with `hasClass`